.rubocop.yml

```diff
@@ -0,0 +1,31 @@
+inherit_gem:
```

Comment:
maybe also document what rubocop version you use ( and probably point do docker file where it is defined ).

Reply:
The Rubocop version is defined by the dependencies of `scc-codestyle` gem, so it's implicitly defined by the version of `scc-codestyle` that you have installed.
I'll add a comment about this though :+1: